### PR TITLE
Update Remotedesktopmanager

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

**Comment by Carlgo11** [comment]
@40504 if you're planning on submitting more pull requests please reset the master branch you're building from. The old and unrelated commits cause some problems when reviewing.
Thanks! 😊

**Comment by 40504** [comment]
ok 👍

File: entries/r/remotedesktopmanager.com.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "tfa": [
       "totp"
     ],
-    "documentation": "https://help.remotedesktopmanager.com/howto_googleauthenticator.htm",
+    "documentation": "https://helpmac.remotedesktopmanager.com/google_authenticator.html",
```
Comment (Carlgo11) [inline entries/r/remotedesktopmanager.com.json:8]:
```suggestion
    "documentation": "https://help.remotedesktopmanager.com/datasources_2factorauthentication.html",
```

File: entries/r/remotedesktopmanager.com.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "tfa": [
       "totp"
     ],
```
Comment (Carlgo11) [inline entries/r/remotedesktopmanager.com.json:7]: Looks like multiple 2FA-methods are available:
```suggestion
      "totp",
      "custom-software",
      "custom-hardware"
    ],
    "custom-software": [
      "Duo"
    ],
    "custom-hardware": [
      "Yubico OTP"
    ],
```

File: entries/r/remotedesktopmanager.com.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
...
-    "documentation": "https://help.remotedesktopmanager.com/howto_googleauthenticator.htm",
+    "documentation": "https://helpmac.remotedesktopmanager.com/google_authenticator.html",
     "notes": "2FA is only available on the Enterprise Edition",
```
Comment (Carlgo11) [inline entries/r/remotedesktopmanager.com.json:9]:
```suggestion
    "notes": "2FA is only supported on Devolutions Server, Microsoft Azure SQL, Microsoft SQL Server, SQLite & SQL Server",
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by Carlgo11** [comment]
Closing due to inactivity.
> **Note**
> The PR author you can still commit the requested changes and re-open this PR.
> If someone else wants to commit the requested changes you can create a new PR and link to this PR.

**Pull request closed without merging**